pkg/storage/driver/labels_test.go

```diff
@@ -47,3 +48,13 @@ func TestLabelsMatch(t *testing.T) {
 		}
 	}
 }
+
+func TestLabelsKeys(t *testing.T) {
+	var test = labels(map[string]string{"KEY_A": "VAL_A", "KEY_B": "VAL_B"})
+	var expect = []string{"KEY_A", "KEY_B"}
```

Comment:
The order of iteration over map keys is undefined. The test therefore is inconsistent. When I ran the test locally, this was the output:
```
=== RUN   TestLabelsKeys
--- FAIL: TestLabelsKeys (0.00s)
    labels_test.go:58: Expected [KEY_A KEY_B], got [KEY_B KEY_A]
```

You could sort the keys returned before testing.

Reply:
Updated， PTAL